examples/tuning/breast_cancer.py

```diff
@@ -0,0 +1,221 @@
+            if key in ("epoch", "fold"):
+                continue
+            value = getattr(signal, key)
+            if value is not None:
+                net.append_history(value, key)
+
+def breast_cancer(x_train, y_train, x_val, y_val, params):
+    print("Iteration parameters: ", params)
+    def weights_init_uniform_rule(m):
+        classname = m.__class__.__name__
+        if classname.find('Linear') != -1:
+            n = m.in_features
+            y = 1.0 / np.sqrt(n)
+            m.weight.data.uniform_(-y, y)
+            m.bias.data.fill_(0)
+    manager = DataManager.from_numpy(
+        train_inputs=x_train, train_labels=y_train,
+        batch_size=params["batch_size"], validation_inputs=x_val,
+        validation_labels=y_val)
+    net = BreastCancerNet(
+        n_feature=x_train.shape[1], first_neuron=params["first_neuron"],
+        second_neuron=params["second_neuron"], dropout=params["dropout"])
+    net.apply(weights_init_uniform_rule)
+    net.init_history()
+    model = Classifier(
+        model=net,
+        optimizer_name=params["optimizer_name"],
+        learning_rate=params["learning_rate"],
+        loss_name=params["loss_name"],
+        metrics=["accuracy"])
+    model.add_observer("after_epoch", update_talos_history)
+    model.training(
+        manager=manager,
+        nb_epochs=params["epochs"],
+        checkpointdir=None,
+        fold_index=0,
+        with_validation=True)
+    return net, net.parameters()
+
+#############################################################################
+# Setting the Parameter Space Boundaries
+# --------------------------------------
+#
+# In the last and final step, we're going to create the dictionary, which will
+# then be passed on to Talos together with the model above. Here we have
+# three different ways to input values:
+# - as stepped ranges (min, max, steps)
+# - as multiple values [in a list]
+# - as a single value [in a list]
+# For values we don't want to use, it's ok to set it as None.
+
+params = {
+    "first_neuron": [200, 100],
+    "second_neuron": [30, 50],
+    "dropout": [0.2, 0.3],
+    "optimizer_name": ["SGD", "Adam"],
+    "loss_name": ["CrossEntropyLoss"],
+    "learning_rate": [1e-3, 1e-4],
+    "batch_size": [20, 50, 5],
+    "epochs": [10, 20]
+}
+
+#############################################################################
+# Run the Hyperparameter scan
+# ---------------------------
+#
+# Now we are ready to run the model based on the parameters and the layer
+# configuration above. The exact same process would apply with any other
+# model, just make sure to pass the model function name in the Scan() command
+# as in the below example. To get started quickly, we're going to invoke only
+# 10 rounds.
+
+scan_object = talos.Scan(x=x_train,
+                         y=y_train,
+                         params=params,
+                         model=breast_cancer,
+                         experiment_name="/tmp/breast_cancer",
```

Comment:
experiment name is just a str which produce an entry in the current dir. The absolute path does not seem to work. 
I propos to replace   experiment_name="/tmp/breast_cancer",
by                             experiment_name="breast_cancer"

and it will create or update the entry breast_cancer locally